package.json

```
@@ -78,11 +80,18 @@
     "eslint-config-standard-with-typescript": "^43.0.0",
     "eslint-plugin-simple-import-sort": "^12.0.0",
     "eslint-plugin-unused-imports": "^3.0.0",
-    "landlubber": "^2.0.0",
+    "eslint-plugin-react": "^7.34.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.4.6",
     "prettier": "^3.0.0",
     "tsc-alias": "^1.8.2",
     "tsup": "^8.0.1",
-    "tsx": "^4.6.2",
-    "typescript": "^5.1.0"
+    "tsx": "^4.7.1",
+    "typescript": "^5.4.2",
```

We may need to do this for now as, if I recall, there is an eslint warning about this TS version being yet-unsupported.
```suggestion
    "typescript": "~5.3.0",
```